tests/network_test.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * This file is part of cc-oci-runtime.
+ *
+ * Copyright (C) 2016 Intel Corporation
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+#include <stdbool.h>
+#include <stdlib.h>
+
+#include <check.h>
+#include <glib.h>
+
+#include "test_common.h"
+#include "../src/oci.h"
+#include "../src/network.h"
+#include "../src/logging.h"
+
+START_TEST(test_cc_oci_vm_pause) {
+	char *socket_path = NULL;
+	pid_t pid = 0;
+	pid = run_qmp_vm (&socket_path);
+	ck_assert (pid > 0);
+	ck_assert (socket_path != NULL);
+
+	ck_assert (! cc_oci_vm_pause (NULL, -1));
+	ck_assert (! cc_oci_vm_pause (socket_path, -1));
+	ck_assert (! cc_oci_vm_pause (NULL, pid));
+	ck_assert (cc_oci_vm_pause (socket_path, pid));
+
+	g_free(socket_path);
```

Comment:
That last call to `cc_oci_vm_pause()` needs to capture `pid` so that you can do `kill (pid, SIGTERM)` to clean up.